api/sessions.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import MetaData, create_engine
+from flask import Flask
+from sqlalchemy.orm import sessionmaker
+
+if __package__ is None or __package__ == '':
+  from settings import DefaultSettings
+else:
+  from .settings import DefaultSettings
+
+app = Flask(__name__)
+app.config.from_object(DefaultSettings)
+
+user= str(app.config['POSTGRES_USER'])
+password= str(app.config['POSTGRES_PASSWORD'])
+host=str(app.config['POSTGRES_HOST'])
+port=str(app.config['POSTGRES_PORT'])
+db=str(app.config['POSTGRES_DB'])
+
+engine = create_engine('postgresql://' + user +':' + password +'@' + host +  ':' + port + '/' + db)
+
+
+class Session():
+    def __init__(self):
+        self.engine = create_engine('postgresql://' + user +':' + password +'@' + host +  ':' + port + '/' + db)
```

Comment:
can we use something like this https://github.com/analyseether/ether_sql/blob/1cda5ee9f6be21b7e84dd7fffc20fb0790ca4b60/ether_sql/session.py#L100 here?

Reply:
Yeah definitely, in fact I tried to look for a "connection string builder" kinda thing for create_engine but couldn't find one. Didn't strike me that it can be done this way as well. Will do this.